### DISCUSSION TITLE
ART 1.15.0

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#2209] [Announcements]
This release of ART 1.15.0 introduces 

# Added

- Added default training function to `TensorFlowV2Classifier` (#2124)
- Added TRADES adversarial training protocol in PyTorch (#2131)
- Added preprocessors for images supporting padding and resizing in PyTorch, TensorFlow and framework-independent (#2138)
- Added support for arbitrarily sized images in `BadDet` poisoning attacks (#2189)
- Added estimator for DEtection TRansformer (DETR) object detection models based on transformer architectures (#2192)

# Changed

- Changed PyTorch estimators to use PyTorch datasets and dataloaders to optimize the `fit` and `predict` methods for `PyTorchClassifier`, `PyTorchRegressor`, `PyTorchRandomizedSmoothing`, `PyTorchObjectDetector`, and `PyTorchYolo`  and optimized the `predict` method of `TensorFlowV2Classifier` by using a TensorFlow dataset and applying @tf.function decorator (#2180)
- Changed `PyTorchObjectDetector` to apply `channels_first` argument  and improved performance by applying batch processing provided by newer PyTorch versions. (#2180)

# Removed

[None] 

# Fixed

- Fixed unnecessary duplicate prediction calls to estimator in `SignOPTAttack` (#2129)
- Fixed missing transfer of tensor to device in `ProjectedGradientDescentPyTorch` (#2135)
- Fixed trigger placement for image poisoning perturbations by correctly accessing height and width of the trigger image instead of swapping both (#2143)
- Fixed key error in loss gradients of `PyTorchYolo` estimator and updated format of targets passed to the estimator in `AdversarialPatchPyTorch` to reflect updates to `PyTorchYolo`(#2169)
- Fixed Visible Deprecation Warning in `analyze_by_distance` and `analyze_by_size` of `ClusteringAnalyzer` (#2195)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.15.0'>ART 1.15.0</a>.</em>
